clients/csrf/forbidden_error.go

```diff
@@ -0,0 +1,12 @@
+package csrf
+
+const RetryIsNeeded = "retry is needed"
+
+type ForbiddenError struct {
```

Comment:
Do not understand at all.

Reply:
Custom error. Thrown when retry is needed.